test/scripts/update-kindnode-version.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Authors of spidernet-io
+# SPDX-License-Identifier: Apache-2.0
+
+echo "Updating k8s matrix run file..."
+KIND_NODE_VERSION=""
+function update_kindnode_version() {
```

Comment:
getAllLatestVersion

there is no update operation in the function

Reply:
done